src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/GaugeWave.hpp

```diff
@@ -91,12 +91,13 @@ namespace Solutions {
  *  - Amplitude (default: 1.)
  *  - Wavelength (default: 1.)
  */
+template <size_t Dim>
 class GaugeWave : public MarkAsAnalyticSolution {
   template <typename DataType>
   struct IntermediateVars;
 
  public:
-  static constexpr size_t volume_dim = 3;
+  static constexpr size_t volume_dim = Dim;
```

Comment:
Why have volume_dim and Dim?

Reply:
WrappedGr reads volume_dim to determine the solution dimension.